### PR TITLE
Misc docs improvements

### PR DESCRIPTION
**Pull request by dannflor**
- Fixed typo in Leaf overview
- Updated version numbers in macOS install guide
- Added warning about Xcode Command Line Tools, closes https://github.com/vapor/docs/issues/355
- Added tip about Vapor Github Templates

### CODE REVIEW

**Comment by dannflor** [comment]
I also wanted to try to close https://github.com/vapor/docs/issues/394 but I couldn't find an appropriate place to put the information. I'd like some guidance on where you think that ought to go, or else we should close it as wontfix.

**Comment by gwynne** [comment]
My suggestion would be to add a "Requests" section under "Advanced" and see if you can find any additional bits that'd go well on the same page.

File: docs/getting-started/hello-world.md [hunk 1]
```diff
@@ -49,6 +52,14 @@ You should see the Console pop up at the bottom of the Xcode window.
...
 [ INFO ] Server starting on http://127.0.0.1:8080
 ```
+!!! warning
+	If you get an error along these lines:
+	```sh
+	error: terminated(72): xcrun --sdk macosx --find xctest output:
+	# or
+	Error: Could not generate Xcode project: error: terminated(72): xcrun --sdk macosx --find xctest output:
+	```
+	You need to run `sudo xcode-select -s /Applications/Xcode.app` to use the correct version of Xcode CLI tools.
```
Comment (gwynne) [inline docs/getting-started/hello-world.md:62]: This information is not relevant to current versions of Swift and Xcode.

File: docs/install/macos.md [hunk 2]
```diff
@@ -19,11 +19,11 @@ swift --version
 You should see Swift's version information printed.
 
 ```sh
-Apple Swift version 5.2 (swiftlang-1100.0.270.13 clang-1100.0.33.7)
-Target: x86_64-apple-darwin19.0.0
+swift-driver version: 1.75.2 Apple Swift version 5.8 (swiftlang-5.8.0.124.2 clang-1403.0.22.11.100)
+Target: x86_64-apple-macosx13.0
```
Comment (gwynne) [inline docs/install/macos.md:23]:
```suggestion
Target: arm64-apple-macosx13.0
```
M1 is more and more common these days.

File: docs/install/macos.md
Comment (0xTim) [inline docs/install/macos.md:1]: Would it be possible to update the translations of this file as well?

Comment (dannflor) [reply docs/install/macos.md:1]: Done

**Review by 0xTim — APPROVED** [review]
Thanks!

**Pull request merged**